packages/element/src/test/create-interpolate-element.js

```diff
@@ -9,12 +9,14 @@ import TestRenderer, { act } from 'react-test-renderer';
 import { createElement, Fragment, Component } from '../react';
 import createInterpolateElement from '../create-interpolate-element';
 
+const conversionError = /The conversionMap provided is not valid./;
+
 describe( 'createInterpolateElement', () => {
-	it( 'returns same string when there is no conversion map', () => {
+	it( 'throws an error when there is no conversion map', () => {
 		const testString = 'This is a string';
 		expect(
-			createInterpolateElement( testString, [] )
-		).toBe( 'This is a string' );
+			() => createInterpolateElement( testString, {} )
+		).toThrow( conversionError );
```

Comment:
just a note: by writing our test to enforce the specific string returned in the error we're saying that the tests will fail if we intentionally change the error message. that may or may not encode too specific of a behavior when maybe the most important thing we want to communicate to people about this function is that it _throws_ or that it throws a specific _type_ of exception.

if we test a hard-coded string then people will rely on a hard-coded string.

Reply:
Fixed in 17f8514